mmv1/third_party/terraform/services/logging/resource_logging_bucket_config_test.go

```diff
@@ -123,42 +123,6 @@ func TestAccLoggingBucketConfigProject_analyticsEnabled(t *testing.T) {
 	})
 }
 
-func TestAccLoggingBucketConfigProject_locked(t *testing.T) {
-	t.Parallel()
-
-	context := map[string]interface{}{
-		"random_suffix":   acctest.RandString(t, 10),
-		"project_name":    "tf-test-" + acctest.RandString(t, 10),
-		"org_id":          envvar.GetTestOrgFromEnv(t),
-		"billing_account": envvar.GetTestBillingAccountFromEnv(t),
-	}
-
-	acctest.VcrTest(t, resource.TestCase{
-		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
-		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
-		Steps: []resource.TestStep{
-			{
-				Config: testAccLoggingBucketConfigProject_locked(context, false),
```

Comment:
Could we instead just switch this line with L150 ? That way the bucket would start out locked then be updated to unlocked before being deleted.

Reply:
Seems we can't, because locking a bucket is irreversible:
https://cloud.google.com/logging/docs/buckets#locking-logs-buckets

Reply:
Understood.